pyomo/contrib/doe/documentation/docs/source/index.rst

```diff
@@ -0,0 +1,453 @@
+############
+Pyomo.DOE
+############
+
+Pyomo.DOE
+===================================
+
+**Pyomo.DOE** (Pyomo Design of Experiments) is a Python library for model-based design of experiments
+using science-based models.
+
+It was developed by **Jialu Wang** and **Alexander W. Dowling** of University of Notre Dame as part of the Carbon Capture Simulation for Industry Impact (CCSI2)
+(https://www.acceleratecarboncapture.org/ ) project, 
+funded through the U.S. DOE Office of Fossil Energy by the Lawrence Berkeley National Laboratory through contract
+DE-AC02-05CH11231 and the National Energy Technology Laboratory, an agency of the United States Government, through a support contract.
+
+If you use Pyomo.DOE, please cite:
+
+[Wang and Dowling, 2022] Jialu Wang, Alexander Dowling (2022), Pyomo.DOE: An Open-Source Package for Model-Based Design of Experiments in Python. (under review)
+
+Methodology Overview
+---------------------
+
+Model-based Design of Experiments (MBDoE) is a technique to maximize the information gain of experiments by directly using
+science-based models with physically meaningful parameters. It is one key component in the model calibration and uncertainty quantification workflow
+shown below:
+
+.. figure:: flowchart.png
+   :scale: 25 %
+   :alt: map to buried treasure
+
+   The exploratory analysis, parameter estimation, uncertainty analysis, and MBDoE are combined into an iterative framework to select, refine, and calibrate science-based mathematical models with quantified uncertainty. The goal of this workflow is to achieve a confident parameter estimation.
```

Comment:
Replace "The goal... parameter estimation" with Currently, Pyomo.DOE focused on increasing parameter precision.